README.md

```diff
@@ -1 +1 @@
-# mev-mantis
+## Etherscan API consumption in Golang
```

Comment:
good to have changed the name of the current project. could you also add commands to:
- install
- compile
- run

Reply:
I updated the README.md as follows: navigate to the cloned folder, then run: `go run extract.go` 
Not going to work though until i implement `main()`